backend/vouch/trust_algo.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from vouch.models import Voucher
+from core.models import user
+
+# hyper parameter of algorithm - select them wisely
+a = 0.2  # define the minimum ratio of trust that is left to pre-trusted users
+eps = 0.00000001
```

Comment:
can be written
```py
eps = 0.000_000_01
```
or simply
```py
eps = 1e-8
```

Reply:
Should we also use upper case name for these parameters, to respect the Python PEP8 style guide?

See: https://peps.python.org/pep-0008/#constants